server/session.py

```diff
@@ -553,10 +553,15 @@ def notify(self, height, touched):
         '''Notify the client about changes in masternode list.'''
         result = super().notify(height, touched)
 
-        for masternode in self.mns:
-            status = self.daemon.masternode_list(['status', masternode])
-            self.send_notification('masternode.subscribe',
-                                   [masternode, status.get(masternode)])
+        try:
+            for masternode in self.mns:
+                status = self.daemon.masternode_list(['status', masternode])
+                self.send_notification('masternode.subscribe',
+                                       [masternode, status.get(masternode)])
+        except DaemonError as e:
+            error, = e.args
+            message = error['message']
+            self.logger.info('there was an error when notifying the masternode status: {}'.format(message))
```

Comment:
I added this try/catch block because sometimes I got this error:

```
electrumx_server.py[13104]: ERROR:Controller:Traceback (most recent call last):
electrumx_server.py[13104]:   File "/usr/local/lib/python3.6/dist-packages/electrumx-1.3.1-py3.6.egg/server/controller.py", line 178, in on_future_done
electrumx_server.py[13104]:     future.result()
electrumx_server.py[13104]:   File "/usr/local/lib/python3.6/dist-packages/electrumx-1.3.1-py3.6.egg/server/block_processor.py", line 206, in main_loop
electrumx_server.py[13104]:     await task()
electrumx_server.py[13104]:   File "/usr/local/lib/python3.6/dist-packages/electrumx-1.3.1-py3.6.egg/server/block_processor.py", line 256, in check_and_advance_blocks
electrumx_server.py[13104]:     self.controller.mempool.on_new_block(self.touched)
electrumx_server.py[13104]:   File "/usr/local/lib/python3.6/dist-packages/electrumx-1.3.1-py3.6.egg/server/mempool.py", line 194, in on_new_block
electrumx_server.py[13104]:     self.controller.notify_sessions(touched)
electrumx_server.py[13104]:   File "/usr/local/lib/python3.6/dist-packages/electrumx-1.3.1-py3.6.egg/server/controller.py", line 296, in notify_sessions
electrumx_server.py[13104]:     session_touched = session.notify(height, touched)
electrumx_server.py[13104]:   File "/usr/local/lib/python3.6/dist-packages/electrumx-1.3.1-py3.6.egg/server/session.py", line 608, in notify
electrumx_server.py[13104]:     'result': status.get(masternode),
```
I know it's not the best way but it's just a workaround, I'll try to fix it later.